app/models/poll_page.rb

```diff
@@ -14,6 +18,7 @@ def self.perform(page: nil)
       return false
     end
 
+    # Should text property be defined here?
```

Comment:
Oh, I've noticed that the text field for PageSnapshots does not seem to get saved into the database, and this might be the spot where its should go.